resources/views/training/training.blade.php

```diff
@@ -0,0 +1,54 @@
+@extends('layouts.layout')
+
+@section('content')
+
```

Comment:
Met weinig data worden de kolommen heel breed wat niet echt leesbaar is. Misschien kan je hier een maximum breedte aan geven of iets dergelijks.

Reply:
fixed